src/index.test.ts

```diff
@@ -136,7 +136,7 @@ describe('logUser', () => {
         return [, , , , , , 'session-id1'];
       },
     };
-    expect(() => logger.innerLogUser(cf, user)).toThrow(/^Inner fail: Failed$/);
+    expect(() => logger.innerLogUser(cf, user)).toThrow(/^Inner fail: Inner fail: Failed$/);
```

Comment:
Is the exact error message useful in the assert? The other test seems to just grep the word "Failed".

Reply:
It doesn't matter.  'Failed' is the error that comes from the fake Snowplow method higher up.  We can rename it to whatever.